cpp/kiss_icp/pipeline/KissICP.hpp

```diff
@@ -80,8 +77,8 @@ class KissICP {
     // KISS-ICP pipeline modules
     std::vector<Sophus::SE3d> poses_;
     KISSConfig config_;
-    AdaptiveThreshold adaptive_threshold_;
     VoxelHashMap local_map_;
+    AdaptiveThreshold adaptive_threshold_;
```

Comment:
gcc complained about a mismatch between the order here and the initialization lists (`-Wreorder`).

Reply:
lgtm